### PR TITLE
cephfs-shell: fix unecessary usage of to_bytes for file paths

### PR DESCRIPTION
**Pull request by batrick**

### CODE REVIEW

**Comment by batrick** [comment]
@varshar16 ping

**Comment by batrick** [comment]
http://pulpito.ceph.com/pdonnell-2019-06-20_02:01:35-fs-wip-pdonnell-testing-20190619.220343-distro-basic-smithi/

File: src/pybind/cephfs/cephfs.pyx [hunk 1]
```diff
@@ -695,16 +744,17 @@ cdef class LibCephFS(object):
         path = cstr(path, 'path')
         cdef:
             char* _path = path
-            ceph_dir_result *dir_handler
+            ceph_dir_result* handle
```
Comment (varshar16) [inline src/pybind/cephfs/cephfs.pyx]: Please change the parameter name in docstrings too.

File: src/tools/cephfs/cephfs-shell [hunk 2]
```diff
@@ -97,40 +97,30 @@ def get_chunks(file_size):
 def to_bytes(string):
     return bytes(string, encoding='utf-8')
 
+def ls(path, opts=''):
+    # opts tries to be like /bin/ls opts
+    almost_all = 'A' in opts
+    with cephfs.opendir(path) as d:
```
Comment (varshar16) [inline src/tools/cephfs/cephfs-shell]: Add try statement before calling opendir.

Comment (batrick) [reply src/tools/cephfs/cephfs-shell]: good point, updated

File: src/pybind/cephfs/cephfs.pyx (hunk 1 above)
Comment (batrick) [reply src/pybind/cephfs/cephfs.pyx]: good catch!

**Pull request merged**